src/Type.cc

```diff
@@ -1981,6 +1981,22 @@ detail::TraversalCode VectorType::Traverse(detail::TraversalCallback* cb) const
 	HANDLE_TC_TYPE_POST(tc);
 	}
 
+TypeManager::TypeManager()
+	{
+	for ( auto i = 0u; i < base_list_types.size(); ++i )
+		{
+		TypeTag tag = TypeTag(i);
```

Comment:
```suggestion
		TypeTag tag{i};
```

Reply:
That didn't work out:
```
../src/Type.cc:1988:25: error: invalid conversion from ‘unsigned int’ to ‘zeek::TypeTag’ [-fpermissive]
 1988 |   TypeTag tag = TypeTag{i};
```

I'm going for a static_cast<TypeTag>(i), or do you have strong opinions for something else?

Reply:
Oh yeah, typetag's an enum isn't it? Ignore that suggestion then.